src/app.js

```diff
@@ -1 +1,20 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { calculateBullsCows } = require('./calculateBullsCows');
+const { generateNumber } = require('./generateNumber');
+const { ioClient } = require('./ioClient');
+
+const numberToGuess = generateNumber();
+let resultOfGuess = '';
+
+while (true) {
+  resultOfGuess = calculateBullsCows(ioClient(), numberToGuess);
+
+  console.log(resultOfGuess);
+
+  if (resultOfGuess === numberToGuess) {
```

Comment:
That looks strange, `calculateBullsCows` returns an object in the format below, but you trying to compare this object with the string `numberToGuess `:
```
const result = {
    bulls: 0,
    cows: 0,
  };
  ```